src/components/greetings/greetings.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { noop } from "../../services/utils";
+
+import createGreetingsLabel from "./greetingsLabel";
+import createBackButton from "./greetingsBackButton";
+
+export default ({ data = {}, onBack = noop } = {}) => {
+  const container = document.createElement("div");
+
+  const label = createGreetingsLabel(data);
+  const button = createBackButton(onBack);
+
+  container.appendChild(button);
```

Comment:
what appedChild is doing inside the react app? Don't use real DOM directly

Reply:
and this